ClearKeep/Scenes/ChatGroup/Views/TagView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  FlexibleView.swift
+//  ClearKeep
+//
+//  Created by đông on 19/04/2022.
+//
+
+import SwiftUI
+import Common
+import CommonUI
+
+private enum Constants {
+	static let paddingTagUser = 8.0
+	static let cornerRadiusTagUser = 80.0
+	static let heightTagUser = 40.0
+}
+
+struct TagView: View {
+	// MARK: - Variables
+	@Binding var groupChatModel: [CreatGroupGetUsersViewModel]
+	@Environment(\.colorScheme) var colorScheme
+	@Environment(\.injected) private var injected: DIContainer
+	var deleteSelect: (CreatGroupGetUsersViewModel) -> Void
+	
+	init(groupChatModel: Binding<[CreatGroupGetUsersViewModel]>, deleteSelect: @escaping (CreatGroupGetUsersViewModel) -> Void) {
+		_groupChatModel = groupChatModel
+		self.deleteSelect = deleteSelect
+	}
+	
+	// MARK: - Body
+	var body: some View {
+		ScrollView(.horizontal, showsIndicators: false) {
+			LazyHStack {
+				ForEach(0..<groupChatModel.count, id: \.self) { index in
+				tagView(for: groupChatModel[index])
+				}
+			}
+			
+		}.frame(height: self.groupChatModel.isEmpty ? 0 : Constants.heightTagUser)
+	}
+
+	private func tagView(for model: CreatGroupGetUsersViewModel) -> some View {
+        Button {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/ChatGroup/Views/TagView.swift#L43"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>TagView.swift:43</code></td>
    </tr>
  </tbody>
</table>